### ISSUE TITLE
lists with role=presentation

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1413]
We got a question about why the following code snippet failed:

```html
<ul role="presentation">
  <li>item 1</li>
  <li>item 2</li>
</ul>
```

To the best of my knowledge, this create listitem nodes that are not part of a list. But in testing this on Safari, Chrome and Firefox, there did not seem to be a problem. We'll need to do some testing to see if IE and Edge behave the same way, but if they do, we should either allow it, or put this into review.

**Comment by Wildebrew** [comment]
The [ARIA 1.1 definition for the presentation role](http://www.w3.org/TR/wai-aria-1.1/#presentation)
states that

> When an explicit or inherited role of presentation is applied to an element with the implicit semantic of a WAI-ARIA role that has [required owned elements](http://www.w3.org/TR/wai-aria-1.1/#mustContain) ... the user agent MUST apply an inherited role of presentation to any owned elements that do not have an explicit role defined. 

The most common use case for the presentation role is to mark a layout table. Simply put `role="presentation"` on the `<table>` element and, voila, you got yourself a layout table.
You do not have to put the presentation role on the `<tr>` <th>` or `<td>` elements) because they are required owned elements for the table.
aXe correctly handles tables with `role="presentation"`.

The same applies to lists. If you put `role="presentation"` on the list container element it inherits to the `<li>` elements, because they are required owned elements of the list container element.

**Comment by dylanb** [comment]
@WilcoFiers @Wildebrew is correct - we need to differentiate in the logic between:

```
<ul role="presentation">
  <li>item 1</li>
  <li>item 2</li>
</ul>
```

Which should pass, and:

```
<ul role="presentation">
  <li role="listitem">item 1</li>
  <li>item 2</li>
</ul>
```

Which should fail

**Comment by WilcoFiers** [comment]
This is a duplicate of #1171, closing it.
